data/json/npcs/missiondef.json

```diff
@@ -1843,5 +1843,16 @@
       "success_lie": "What?!  You're lying, I can tell!  Ugh, forget it!",
       "failure": "Oh no!  My poor puppy..."
     }
+  },
+  {
+    "id": "MISSION_REACH_REFUGEE_CENTER",
+    "type": "mission_definition",
+    "name": "Reach Refugee Center",
+    "goal": "MGOAL_GO_TO_TYPE",
+    "difficulty": 1,
+    "value": 50000,
```

Comment:
Unfortunately, It can't be simply omitted: it's mandatory. We really should split the missions (to avoid hacks like `return origin == ORIGIN_ANY_NPC || origin == ORIGIN_OPENER_NPC || origin == ORIGIN_SECONDARY;`). 
`"value": 0` would suffice for now.